src/components/web/WebApp.tsx

```diff
@@ -521,6 +521,7 @@ const WebApp = ({
             'mailto://*'
           ]}
           cacheEnabled={false}
+          cacheMode='LOAD_NO_CACHE'
```

Comment:
I don't think we need this one. Seems to be android-only related https://github.com/react-native-webview/react-native-webview/blob/404e3e69e23d8c73bc4deb688c77b9a1a0bb55f8/docs/Reference.md#cachemode, unless we have similar issue with android.